modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPRequest.macro

```diff
@@ -115,4 +115,12 @@ definition {
 		MenuItem.click(menuItem = ${key_option});
 	}
 
+	macro viewSuccessMessageRequest {
+		while (IsElementPresent(locator1 = "Message#EMPTY_INFO")) {
+			AssertTextEquals(
+				locator1 = "Message#EMPTY_INFO",
+				value1 = "Thank you. Your information was successfully received.");
+		}
+	}
+
 }
```

Comment:
You can remove this macro, because is equals to the other macro, that will be replaced to EVPUtils